apps/src/codebridge/Workspace/index.tsx

```diff
@@ -4,16 +4,26 @@ import {FileTabs} from '@codebridge/FileTabs';
 import React from 'react';
 
 import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
+import ProjectTemplateWorkspaceIcon from '@cdo/apps/templates/ProjectTemplateWorkspaceIcon';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 
 import HeaderButtons from './HeaderButtons';
 
 import moduleStyles from './workspace.module.scss';
 const Workspace = () => {
   const {config} = useCodebridgeContext();
+  const isProjectTemplateLevel = useAppSelector(
```

Comment:
We actually already have a [isProjectTemplateLevel](https://github.com/code-dot-org/code-dot-org/blob/478de6f89be9d4c3a3c55a0b8fd1e128145d3a3e/apps/src/lab2/lab2Redux.ts#L275) selector in lab2redux that you should be able to use here 🙂

Reply:
thanks! Will update